modules/command-engine/core/src/test/scala/surge/internal/domain/SurgeMessagePipelineSpec.scala

```diff
@@ -25,7 +27,6 @@ import surge.internal.health.windows.stream.sliding.SlidingHealthSignalStreamPro
 import surge.kafka.streams.{ AggregateStateStoreKafkaStreams, MockPartitionTracker, MockState }
 import surge.metrics.Metrics
 
-import java.util.regex.Pattern
 import scala.concurrent.duration._
 
 @Ignore
```

Comment:
Leaving this as ignored for now. Updating the timing has helped a few tests pass a little more reliably & quickly but some still fail with the shorter times.  I'm wondering if it's really just trying to do too many things & needs to be broken down a bit more to test smaller components.

On the few that do pass reliably, there's still an issue where something isn't shut down and the logs for the remainder of testing are flooded with errors connecting to Kafka (since the embedded brokers are spun down after the tests) which makes it pretty challenging to debug any other test failures.  I think before re-enabling this particular suite at least this should be figured out.